resources/discord.py

```diff
@@ -0,0 +1,14 @@
+import pulumi_discord
+
```

Comment:
I am the search on this page, https://www.pulumi.com/docs/.
It doesn't list pulumi_discord, I see `sdks/discord`. Is that what enables this? How do we get that folder?

Reply:
Yep, `sdks/discord` is where this is coming from. Pulumi does not have a Discord provider, but [Terraform does](https://registry.terraform.io/providers/Lucky3028/discord/latest/docs). Pulumi provides a tool ([pulumi-terraform](https://github.com/pulumi/pulumi-terraform)) to convert Terraform providers to Pulumi ones. That's the tool I used to generate the `sdks` directory.

Reply:
Honestly, not sure. the `preview` command was working at least. I was able to run `up` locally, but haven't tried from a GH action. If it fails, I'll dig through and figure out what we need to do.